ananse/peakpredictor.py

```diff
@@ -490,22 +492,25 @@ def _load_data(self, factor):
         # logger.debug(str(self._X_columns))
         return tmp[self._X_columns]
 
-    def _load_model(self, factor):
+    def _load_model(self, factor, jacard_cutoff = 0.0):
         model = None
+        motif_edge_min = 1 - jacard_cutoff
         if factor in self.factor_models:
             logger.info(f"Using {factor} model")
             model = self.factor_models[factor]
         elif factor in self.motif_graph:
+            logger.info(f"checking for alternatif model based on motif overlap")
             paths = {
                 p: v
                 for p, v in nx.single_source_dijkstra_path_length(
-                    self.motif_graph, factor
+                    self.motif_graph, factor, cutoff = motif_edge_min
                 ).items()
                 if p in self.factor_models
             }
             try:
                 sub_factor = list(paths.keys())[0]
-                logger.info(f"Using {factor} motif with {sub_factor} model weights")
+                second_sub_factor = list(paths.keys())[1]
+                logger.info(f"Using {factor} motif with {sub_factor} model weights v:{v} second:{list(paths.keys())[1]}")
```

Comment:
what does this print? o.0

Reply:
Aaah shoot, I thought i removed that. That was me just checking stuff and shouldnt be there